docs/ccbc.disaster.recovery.patroni.txt

```diff
@@ -0,0 +1,134 @@
+========================================================
+CCBC Disaster Recovery Testing with Patroni
+========================================================
+
+Recommendations: 
+    From the Patroni Commands Reference below it is highly recommended that you 
+	run any ONE of the following commands in order to test disaster recovery on crunchy cluster.
+	Note the Dev Team testing was restricted to performing #8 below.
+
+3. patronictl switchover
+7. patronictl restart
```

Comment:
3,7,8 here should be hyperlinks to correspondent commands in the text below to provide context.